starknet_devnet/constants.py

```diff
@@ -19,3 +19,4 @@
 DEFAULT_ACCOUNTS = 10
 DEFAULT_INITIAL_BALANCE = 10**21
 DEFAULT_GAS_PRICE = 10**11
+DEFAULT_TIMEOUT = 60
```

Comment:
Add comment about it being seconds (or change var name)

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/256/commits/b715d1628e248a90cc36c49fd6aa78c9d117f1fb